src/io/ImportPhysicsVector.hh

```diff
@@ -4,7 +4,7 @@
 // SPDX-License-Identifier: (Apache-2.0 OR MIT)
 //---------------------------------------------------------------------------//
 //! \file ImportPhysicsVector.hh
-//! \brief Store Geant4 physics vector data
+//! Store Geant4 physics vector data.
```

Comment:
I suggest removing the term "Geant4" for consistency. I changed the comments when refactoring the names and missed this one.

```suggestion
//! Store physics vector data.
```